usaspending_api/recipient/delta_models/sam_recipient.py

```diff
@@ -1,29 +1,107 @@
     """
+sam_recipient_load_sql_string = fr"""
+-- Create temp table to reload into the duns table without dropping the destination table
+DROP TABLE IF EXISTS int.sam_recipient;
+
+CREATE TABLE int.sam_recipient AS (
+  SELECT
+    broker_SR.awardee_or_recipient_uniqu AS awardee_or_recipient_uniqu,
+    broker_SR.legal_business_name AS legal_business_name,
+    broker_SR.ultimate_parent_unique_ide AS ultimate_parent_unique_ide,
+    broker_SR.ultimate_parent_legal_enti AS ultimate_parent_legal_enti,
+    broker_SR.sam_recipient_id AS broker_duns_id,
+    broker_SR.record_date AS update_date,
+    broker_SR.address_line_1 AS address_line_1,
+    broker_SR.address_line_2 AS address_line_2,
+    broker_SR.city AS city,
+    broker_SR.congressional_district AS congressional_district,
+    broker_SR.country_code AS country_code,
+    broker_SR.state AS state,
+    broker_SR.zip AS zip,
+    broker_SR.zip4 AS zip4,
+    broker_SR.business_types_codes AS business_types_codes,
+    broker_SR.dba_name as dba_name,
+    broker_SR.entity_structure as entity_structure,
+    broker_SR.uei AS uei,
+    broker_SR.ultimate_parent_uei AS ultimate_parent_uei
+  FROM
+    global_temp.broker_SR as broker_SR
+      SELECT
+        DISTINCT ON (broker_SR.awardee_or_recipient_uniqu)
+        broker_SR.awardee_or_recipient_uniqu,
+        broker_SR.legal_business_name,
+        broker_SR.dba_name,
+        broker_SR.ultimate_parent_unique_ide,
+        broker_SR.ultimate_parent_legal_enti,
+        broker_SR.address_line_1,
+        broker_SR.address_line_2,
+        broker_SR.city,
+        broker_SR.state,
+        broker_SR.zip,
+        broker_SR.zip4,
+        broker_SR.country_code,
+        broker_SR.congressional_district,
+        COALESCE(broker_SR.business_types_codes, text[]) AS business_types_codes,
+        broker_SR.entity_structure,
+        broker_SR.sam_recipient_id,
+        COALESCE(broker_SR.activation_date, broker_SR.deactivation_date) as record_date,
+        broker_SR.uei,
+        broker_SR.ultimate_parent_uei
+      FROM
+        global_temp.sam_recipient as broker_SR
+      ORDER BY
+        broker_SR.awardee_or_recipient_uniqu,
+        broker_SR.activation_date DESC NULLS LAST)') AS broker_duns
+          (
+            awardee_or_recipient_uniqu text,
+            legal_business_name text,
+            dba_name text,
+            ultimate_parent_unique_ide text,
+            ultimate_parent_legal_enti text,
+            address_line_1 text,
+            address_line_2 text,
+            city text,
+            state text,
+            zip text,
+            zip4 text,
+            country_code text,
+            congressional_district text,
+            business_types_codes text[],
+            entity_structure text,
+            sam_recipient_id text,
+            record_date date,
+            uei text,
+            ultimate_parent_uei text
+          )
```

Comment:
Is this the leftover postgres `dblink` style query? I'm pretty sure we're getting away from that completely here, and querying the broker DB directly, rather than querying it through the usaspending db over `dblink`. 

It looks like maybe this SQL is just a placeholder for the ultimate Spark SQL? 
You'll want some unit tests that exercise whatever SQL you make to ensure it does in fact work. That will be a challenge to get the broker<->usaspending integration tests working just right. 
I'd say
1. Fix the sql
2. Point your local env at a remote broker DB (e.g. sandbox) while on VPN
3. Test the `load_query_to_delta` locally with something like: `make localhost-spark-submit django_command="load_query_to_delta --destination-table int.sam_recipient"`
4. Ensure you see the data you want in your local delta tables by opening up `make pyspark-shell` (you'll need to pull qat branch to get this) and then run `spark.sql("select * from int.sam_recipient").show()`
5. Then work on getting your own local broker DB setup in docker (it doesn't need data)
6. Then ensure you have the right local config to run broker integration tests (Daniel or I could help with this)
7. Then write some unit tests that inject some `data_broker.public.sam_recipient` data into the postgres DB, and then run your command to ensure it is working across DBs.